### DISCUSSION TITLE
Price Feed Version Mismatch: Test Failing on Ethereum Mainnet but Passing on Sepolia Testnet

### DISCUSSION THREAD

**Discussion by tanishqjotwani** [Cyfrin/foundry-full-course-cu#3071] [General]
Hi, I'm facing an issue with one of my tests while deploying and testing my FundMe contract. The test passes without any errors on the Sepolia testnet but fails on the Ethereum mainnet due to a mismatch in the expected price feed version.

```
[FAIL: assertion failed: 6 != 4] testPricefeedVersionIsAccurate() (gas: 18896)
```

**Comment by EngrPips** [comment]
Hello @tanishqjotwani, Yeah, the version was updated to `6` on Mainnet I believe, Sorry for the confusion.

**Reply by tanishqjotwani** [reply]
Ohh , okay. Thank you for answering:)

**Reply by EngrPips** [reply]
You are welcome.

**Comment by AnaAse** [comment]
I replaced:    assertEq(version, 4);

with:             assertTrue(version == 4 || version == 6, "Version must be 4 or 6");    

It works and it will accept either 4 or 6 but Im not sure where these two version numbers come from.

**Reply by EngrPips** [reply]
It is only of these values `6 & 4` that is correct, and it comes from the price feed contract, which you called `getVersion` or `version`. Because you used a logical `or`, the test passes if any of the conditions is satisfied.

**Reply by chainflash** [reply]
great
